api/base/controllers/roles.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+var simpleDI = require('config/simpleDI');
+
+module.exports = simpleDI.inject(['mongoose', 'base/roleModel', 'jsonwebtoken', 'app/config'], function (mongoose, Role, jwt, appConfig) {
+
+  var ObjectId = mongoose.Types.ObjectId;
+
+  return {
+
+    /**
+     * Create role
+     * requires: {roleName}
+     * returns: {message}
+     */
+      create: function (req, res, next) {
+          var newRole = new Role();
+          newRole.roleName = req.body.roleName;
+          try {

```

Comment:
There's no need to use try-catch blocks because you're passing a callback to handle errors when saving the model.